libs/utils/src/env.rs

```diff
@@ -64,7 +65,7 @@ pub fn get_deployment_mode() -> Option<DeploymentMode> {
             }
         },
         Err(_) => {
-            tracing::error!("DEPLOYMENT_MODE not set");
+            // tracing::error!("DEPLOYMENT_MODE not set");
```

Comment:
keep this?

Reply:
Keeping this will fail a lot of tests due to unexpected errors since we are not setting deployment mode.